README.md

```diff
@@ -66,15 +66,20 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
+* Create a kubeconfig file which has the context of all the clusters. If  KUBECONFIG environment variable set with the path to all clusters, you could use the below command.
+```
+kubectl config view --flatten > kubeconfig_merged
+```
+
 * Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
 ```
-subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME> --clusterid  <CLUSTER-ID-FOR-TUNNELS>
+subctl deploy-broker --kubecontext <BROKER-CONTEXT-NAME>  --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME> --clusterid  <CLUSTER-ID-FOR-TUNNELS>
```

Comment:
Add a warning to highlight that due to a bug deploy-broker requires unmerged kubeconfg and should use workaround of using unmerged config.

> :warning: Due to a bug `deploy-broker` fails if using merged kubeconfig. Use unmerged broker kubeconfig as a workaround while issue is being fixed.

Add link to issue.

Reply:
I think since it is due to Kubefed and removal of kubefed will fix this, it would be ok right if we don't mention an explicit warning. May be general warning in the initial section should be fine.